app/serializers/project_serializer.rb

```diff
@@ -37,7 +37,7 @@ class ProjectSerializer
     subjects_export: { include: false },
     aggregations_export: { include: false }
   can_filter_by :display_name, :slug, :beta_requested, :beta_approved,
-    :launch_requested, :launch_approved, :private, :state, :live
+    :launch_requested, :launch_approved, :private, :state, :live, :organization_id
```

Comment:
you get FK filtering for free via restpack, https://github.com/RestPack/restpack_serializer#filtering so no need for the explicit setup